exercises/js/navigation.js

```diff
@@ -0,0 +1,34 @@
+// let ul = $("#nav");
+// ul.hover(function (event) {
+//   let target = $(event.target.closest("li"));
+//   if(target.length && target.is("[data-class='main_menu']")) {
+    
+//   }
+// });
+
+class DropDownMenu {
+  constructor(mainMenuSelector) {
+    this.mainMenu = $(mainMenuSelector);
+  }
+
+  init() {
```

Comment:
method is too long.
create methods and use *init* method to call those methods.